### PR TITLE
[ios] fix compass font thinner than previous version on iOS 13

### PR DESCRIPTION
**Pull request by m-stephen**
close: #14831

It seems that `UIFontWeightLight` on iOS13 is as thin as `UIFontWeightUltraLight` on iOS12.
This is a solution for fixing this issue.

We can wait to merge this PR until iOS13 released and let's see Apple could fix this issue or not.

### CODE REVIEW

File: platform/ios/src/MGLCompassButton.mm [hunk 1]
```diff
@@ -63,9 +63,17 @@ - (UIImage *)compassImage {
     UIImage *scaleImage = [UIImage mgl_resourceImageNamed:@"Compass"];
     UIGraphicsBeginImageContextWithOptions(scaleImage.size, NO, UIScreen.mainScreen.scale);
     [scaleImage drawInRect:{CGPointZero, scaleImage.size}];
+    
+    UIFont *northFont;
+    if(@available(iOS 13, *)){
```
Comment (friedbunny) [inline platform/ios/src/MGLCompassButton.mm]:
```suggestion
    if (@available(iOS 13.0, *)) {
```

File: platform/ios/src/MGLCompassButton.mm [hunk 2]
```diff
@@ -63,9 +63,17 @@ - (UIImage *)compassImage {
...
+        northFont = [UIFont systemFontOfSize:11 weight:UIFontWeightLight];
+    }
+    else{
```
Comment (friedbunny) [inline platform/ios/src/MGLCompassButton.mm]:
```suggestion
    } else {
```

File: platform/ios/src/MGLCompassButton.mm [hunk 3]
```diff
@@ -63,9 +63,17 @@ - (UIImage *)compassImage {
     UIImage *scaleImage = [UIImage mgl_resourceImageNamed:@"Compass"];
     UIGraphicsBeginImageContextWithOptions(scaleImage.size, NO, UIScreen.mainScreen.scale);
     [scaleImage drawInRect:{CGPointZero, scaleImage.size}];
+    
+    UIFont *northFont;
+    if(@available(iOS 13, *)){
+        northFont = [UIFont systemFontOfSize:11 weight:UIFontWeightLight];
+    }
```
Comment (friedbunny) [inline platform/ios/src/MGLCompassButton.mm]: It’s not possible to suggest multiline edits on GitHub, so this line gets deleted (but is included in another suggested edit).
```suggestion
```

**Comment by m-stephen** [comment]
Update for screenshots:
iOS12:
image

Before this PR on iOS13:
image


After this PR on iOS13:
image

**Pull request merged**